src/client/styles/pie.scss

```diff
@@ -16,24 +16,86 @@
  *  limitations under the License
  *
  */
-pie-item {
-  display: flex;
-}
+ 
+.pie-card {
+  border: 1px solid #ccc;
+  position: relative;
+  box-sizing: border-box;
+  margin: 12px;
+  display: inline-block;
 
-pie-img {
-  background: url(/pieimg/pie-svg) center center/cover no-repeat;
-  width: 300px;
-  height: 200px;
-  transition: background .2s;
-}
+  &.pie-card--small {
+    width: 220px;
+  }
 
-.pie-title {
-  min-width: 200px;
-  min-height: 28px;
-  background-position: 0 100%;
-  transition: background-position .2s;
-}
-.pie-description {
-  min-height: 200px;
-  min-width: 300px;
+  &.pie-card--mobile {
+    width: 320px;
+
+    .pie-card-top {
+      display: flex;
+
+      & > pie-img,
+      & > div {
```

Comment:
This resolves to `.pie-card.pie-card--mobile .pie-card-top > div`. It's a bit of a code smell to target a native HTML element inside a nicely named structure like the rest of this.

Perhaps the `div` should get a class as well and we can target that instead, opening up the possibility of using other HTML elements too? Or is there a good reason for making this specific to `div`s at that particular place in the DOM?